src/containers/ViewImageContainer.js

```diff
@@ -0,0 +1,22 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
```

Comment:
As far as I can recall, I tried to do this similarly to how WhatsApp does it, which is just the image on white background and no visible modal. And so I thought it might be easiest to do it this way rather than changing the styles on modal. Is that bad..? I've no clue 😄